sensor_stack/nqDVL/src/nqDVL.py

```diff
@@ -145,26 +145,34 @@ def getVal(rawdata):
 #                print 'got it'
 #            print k
 #        print 'out'
-	
-	r = rospy.Rate(5)
+
+	if rospy.has_param('/ros_rate'):
+		temp_rate = rospy.get_param('/ros_rate')
+	else:
+		raise RuntimeError("ROSParam '/ros_rate' does not exist.")
+		rospy.signal_shutdown("ROSParam '/ros_rate' does not exist.")
+		sys.exit(0)
+
+	r = rospy.Rate(temp_rate)
+
```

Comment:
@aniket11bh @kshitijgoel007 @VishnuDuttSharma We should change these rates to the global constant right?